examples/AWS IoT/AWS_IoT_Opta/AWS_IoT_Opta_WiFi/AWS_IoT_Opta_WiFi.ino

```diff
@@ -0,0 +1,247 @@
+#include <ArduinoBearSSL.h>
+#include <ArduinoECCX08.h>
+#include <ArduinoMqttClient.h>
+#include <Arduino_ConnectionHandler.h>
+#include <Arduino_JSON.h>
+#include <NTPClient.h>
+#include <mbed_mktime.h>
+
+#include "arduino_secrets.h"
+
+// Enter your sensitive data in arduino_secrets.h
+constexpr char broker[] { SECRET_BROKER };
+constexpr unsigned port { SECRET_PORT };
+const char* certificate { SECRET_CERTIFICATE };
+
+// Remove comment for using the Ethernet connection
+// #define USE_ETHERNET_CONNECTION
+
+#if defined(USE_ETHERNET_CONNECTION)
+#include <Ethernet.h>
+#include <EthernetUdp.h>
+EthernetConnectionHandler conMan;
+EthernetClient tcpClient;
+EthernetUDP NTPUdp;
+#else
```

Comment:
If the example is WiFi only i would remove the Ethernet variables, or we can add one single example working both for ethernet and WiFi and name it  `AWS_IoT_Opta.ino`

Reply:
Ethernet configuration removed